src/pages/LandingPage/LandingPage.js

```diff
@@ -45,8 +45,8 @@ class LandingPage extends Component {
           <h3 className="text-2xl font-semibold text-indigo-700 font-serif mt-8 mb-4">
             Rooms
           </h3>
-          <div className="block w-full">
-            <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+          <div className="flex flex-wrap">
+          <div className="w-full sm:w-1/2 md:w-1/3 lg:w-1/4 xl:w-1/6 mb-4" >
```

Comment:
```suggestion
          <div className="w-full sm:w-1/2 md:w-1/3 lg:w-1/4 xl:w-1/6 mb-4 sm:mr-4" >
```

This will just give a bit of space to the right of this card when viewed on a device above the `small` breakpoint